out/cloud_foundry.go

```diff
@@ -36,12 +36,21 @@ func (cf *CloudFoundry) Target(organization string, space string) error {
 }
 
 func (cf *CloudFoundry) PushApp(manifest string, path string, currentAppName string) error {
+	zeroDowntimeDeploy := currentAppName != ""
+
+	if zeroDowntimeDeploy {
+		err := cf.cf("check-manifest", currentAppName, "-f ", manifest).Run()
```

Comment:
Removed.

Funny that the tests pass both with and without it. Something must be stripping them at a lower level.